packages/test/harness/.eslintrc.yml

```diff
@@ -0,0 +1,17 @@
+env:
```

Comment:
do you have to add ignore globals: process in this file?

Reply:
We don't need to. `env.node` is set to `true`, which implies globals from Node.js will automatically applied, including `process`.